tests/lib/cli-engine/cli-engine.js

```diff
@@ -4647,6 +4647,18 @@ describe("CLIEngine", () => {
             }, `There was a problem loading formatter: ${fullFormatterPath}\nError: Cannot find module '${fullFormatterPath}'`);
         });
 
+        it("should return null when a built-in formatter no longer exists", () => {
```

Comment:
```suggestion
        it("should throw an error when a built-in formatter no longer exists", () => {
```

?

Reply:
Based on the other cases, I'd say `throw` rather than `throw an error`, @lydell.

Reply:
```suggestion
        it("should throw when a built-in formatter no longer exists", () => {
```